.github/actions/update-docs/action.yml

```diff
@@ -47,12 +47,12 @@ runs:
       shell: bash
       if: ${{ inputs.release == 'false' }}
       run: |
-        poetry run mike deploy ${{ inputs.version }} --push --rebase --config-file ./docs/mkdocs.yml
+        poetry run mike deploy ${{ inputs.version }} --push --config-file ./docs/mkdocs.yml
```

Comment:
does docs publishing from PRs and main branch still work?

Reply:
[Doc publishing from PRs](https://github.com/bakdata/kpops/commit/d1087cfdf422191c3c07913ad7c0bd669b2ce623)

[Doc publishing from main](https://bakdata.github.io/github-playground-2/main/user/changelog/)

[Doc publishing on release](https://bakdata.github.io/github-playground-2/3.2/user/changelog/)